tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2965,7 +2993,13 @@ def reducer(x):
 
 
 @tf_export('image.ssim')
-def ssim(img1, img2, max_val):
+def ssim(img1,
+         img2,
+         max_val,
+         filter_size=_SSIM_FILTER_SIZE,
```

Comment:
Same here, the actual values make the code more self-documenting

Reply:
Also the docstring should be rewritten to mention that the values for filter_sigma, k1, and k2 are only default (line 3015)